Jenkinsfile

```diff
@@ -7,18 +7,7 @@ pipeline {
         UACLIENT_BEHAVE_JENKINS_CHANGE_ID = "${CHANGE_ID}"
         UACLIENT_BEHAVE_BUILD_PR=1
         UACLIENT_BEHAVE_CONTRACT_TOKEN = credentials('ua-contract-token')
-        UACLIENT_BEHAVE_AWS_ACCESS_KEY_ID = credentials('ua-aws-access-key-id')
-        UACLIENT_BEHAVE_AWS_SECRET_ACCESS_KEY = credentials(
-            'ua-aws-secret-access-key'
-        )
-        UACLIENT_BEHAVE_AZ_CLIENT_ID = credentials('ua-azure-client-id')
-        UACLIENT_BEHAVE_AZ_CLIENT_SECRET = credentials(
-            'ua-azure-client-secret'
-        )
-        UACLIENT_BEHAVE_AZ_TENANT_ID = credentials('ua-azure-tenant')
-        UACLIENT_BEHAVE_AZ_SUBSCRIPTION_ID = credentials(
-            'ua-azure-subscription-id'
-        )
+        UACLIENT_BEHAVE_CLOUD_CREDENTIALS_PATH = credentials('pycloudlib-config')
```

Comment:
Does this need to be established up in UA integration tests? pycloud lib already readys PYCLOUDLIB_CONFIG environment variable. If expected config is absent pycloudlib yells loudly and instructs how to fix it.

Reply:
Yeah, I will make that variable optional. However, I think we will need to setup the `PYCLOUDLIB_CONFIG` env var on jenkins, right @paride ?

Reply:
@lucasmoura yes but it has to be done here in the Jenkinsfile

    PYCLOUDLIB_CONFIG = credentials('pycloudlib-config')`

For this kind of job that's the only way, we can't use `credentials-binding`.

Reply:
Here I'm a bit lost: why do we need `UACLIENT_BEHAVE_CLOUD_CREDENTIALS_PATH` at all given that pycloudlib is going to use `PYCLOUDLIB_CONFIG`? Not suggesting dropping it, but I'm failing to get its use case.